tests/test_fields.py

```diff
@@ -32,3 +32,33 @@ def test_can_parse_quoted_property_names_with_extra_spaces(self):
     def test_can_parse_unquoted_property_names_with_extra_spaces(self):
         node = ' DOCPROPERTY  Text Property  \\* MERGEFORMAT '
         assert "Text Property" == FieldForTesting(node).name
+
+    def test_can_parse_quoted_date_format(self):
+        node = ' DOCPROPERTY "Propertyname" \\@ "dd-yy-mm" \\* MERGEFORMAT '
+        assert "Propertyname" == FieldForTesting(node).name
```

Comment:
Because I thought it was our convention in tests to have first the "expected" value, then the "actual" value. I do agree with you that it is rather less readable though...

Reply:
I guess this comes from JUnit where `expected` is the first argument for the `assertEquals` methods. But in Python this isn't the case. Some people are arguing that you get better test feedback if you follow the JUnit convention. Though I'm not convinced about that because Python's unittest doesn't make this assumption and the [examples](https://docs.python.org/3/library/unittest.html#basic-example) pass the arguments in the "expected" order.

With pytest and `assert` there's definitely no relation to JUnit. Thus I would prefer the more readable variant.
This is not something that has to be fixed in this PR, but we should discuss this in the team and agree to a common style.

Reply:
I'll add a commit and fix it. I quickly discussed with deif and lukas and they agree with you. I did not realise where the convention came from and that it does not apply here.